content/en/docs/reference/kubectl/overview.md

```diff
@@ -91,7 +91,7 @@ Operation       | Syntax    |       Description
 `port-forward`    | `kubectl port-forward POD [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N] [flags]` | Forward one or more local ports to a pod.
 `proxy`        | `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix] [flags]` | Run a proxy to the Kubernetes API server.
 `replace`        | `kubectl replace -f FILENAME` | Replace a resource from a file or stdin.
-`run`        | `kubectl run NAME --image=image [--env="key=value"] [--port=port] [--dry-run=server|client|none] [--overrides=inline-json] [flags]` | Run a specified image on the cluster.
+`run`        | <code>kubectl run NAME --image=image [--env="key=value"] [--port=port] [--dry-run=server &#124; client &#124; none] [--overrides=inline-json] [flags]</code> | Run a specified image on the cluster.
```

Comment:
Hi @viniciusbds .
I noticed that some commands are displaying the flags with a double hyphen (expected)
while other commands show the flag prefixed with a solid line.
Do you see the difference?

Reply:
really @kbhawkey, I will fix this

Reply:
the page contains other parts with the same problem, do I fix them too?

thanks for review :)

Reply:
Hi @kbhawkey
I made some changes, I think it's better now.
Could you review it?